html-css/trending/index.html

```diff
@@ -0,0 +1,173 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>Layout</title>
+    <style>
+        html,
+        body {
+            margin: 0;
+            background: rgb(242, 242, 242);
+            height: 99%;
+        }
+
+        .main-section {
+            display: flex;
+            flex-direction: row;
+            justify-content: center;
+            flex-wrap: wrap;
+            margin: auto;
+            width: 80%
```

Comment:
I would remove all hard-coded margin on .topic and use 
`justify-content: space-between;` in main section.

then .topic will have 
`width: calc(50% - 10px);
    margin: 10px 0;`

wdyt?

Reply:
removed left and right margins but from a quick search it seems flex doesn't offer vertical spacing